ci-operator/config/openshift/ols-load-generator/openshift-ols-load-generator-main.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```

Comment:
@vishnuchalla may be we want to update to latest version ?

Reply:
Done